src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -483,10 +484,27 @@ private void UpdateAdditionalFiles(Project project, IList<string> additionalFile
 
             foreach (var file in additionalFiles)
             {
-                 if (File.Exists(file))
-                 {
+                if (File.Exists(file))
+                {
                     _workspace.AddAdditionalDocument(project.Id, file);
-                 }
+                }
+            }
+        }
+
+        private void UpdateAnalyzerConfigFiles(Project project, IList<string> analyzerConfigFiles)
+        {
+            var currentAnalyzerConfigDocuments = project.AnalyzerConfigDocuments;
```

Comment:
what's the value of `AnalyzerConfigDocuments` if it's not set in the project file explicitly? Should you not fallback here to `EditorConfigFinder`?

Reply:
The AnalyzerConfigDocuments property is computed by the build targets. It is present in build tools for 16.3 and higher, so should be present based on the supported msbuild versions.

Reply:
ah good